osu.Game/Screens/Menu/Button.cs

```diff
@@ -282,10 +282,13 @@ protected override void Update()
 
         public ButtonState State
         {
-            get { return state; }

```

Comment:
Okey i will ignore rule (SA1407) and change back to original reduce form in next commit. Also to prevent this from showing up you can ignore rule (SA1407) in codefactor